src/js/plugins/tts/tooltips.js

```diff
@@ -0,0 +1,15 @@
+export const en = {
```

Comment:
I'd probably name this file as something else bc it's not really tooltips - it's text for them?

Reply:
Yeah, it's a tooltip dictionary. Will rename.